js/apps/events-page/dataServices/eventsService.js

```diff
@@ -82,7 +82,7 @@
 							// Since moment().subtract() mutates the date rather than returning a new date,
 							// we need to calculate the date fresh every time.
 							response.data.registrationStarts = moment(response.data.EventStart).subtract(7, 'days');
-							response.data.registrationEnds = moment(response.data.EventStart).subtract(30, 'minutes');
+							response.data.registrationEnds = moment(response.data.EventStart);
```

Comment:
Can the time offsets for both `registrationStarts` and `registrationEnds` be fed from a configuration file?

Reply:
yes, and it's desired.